### ISSUE TITLE
Not syncing..

### ISSUE THREAD

**Issue by jiwanbeth** [fiddur/fitbit2runkeeper#8]
Tried to get this to work, sadly no syncing appears to be happening.. Is this still a working option?

**Comment by jiwanbeth** [comment]
The 'List fitbit activities' option on your site is also not working

**Comment by fiddur** [comment]
I deployed a new version yesterday that fixed a URL issue with getting fitbit activities consecutive times. It seems to be working well at least after a day; let me know if it works for you as well!

**Comment by jiwanbeth** [comment]
Great! thanks! The only problem is i revoked acces from my runkeeper account, and i can not find a way to real instate the activitysyncer on runkeeper. Is there a way i can do this?

**Comment by fiddur** [comment]
Hmm, perhaps there is a way to reconnect it here: https://runkeeper.com/apps/activitysyncer/799013
Otherwise I might have to manually delete your old account so that it doesn't think you're already connected... (or add a reconnect option)

**Comment by jiwanbeth** [comment]
Sadly that does not work.. I think you have to delete me, so if you could that would be great! A reconnection option would also be nice, but i can imagine that takes time to make

**Comment by fiddur** [comment]
Ok, supposing you are the one with the same name in the db, you are now removed.  You may have to remove cookies or use "private browsing" to be able to connect fitbit and runkeeper again.

**Comment by jiwanbeth** [comment]
It's Working great! Thanks! I have another question though. I bike a lot, thus my biking activities are also syncd to Runkeeper. Only the syncing is not completely working, as at the moment i believe Fibit registers the activity as "Bike" while Runkeeper calls it "Cycling". Therefore on runkeeper it appears as "others". I was wondering if it were possible in the code to translate the "Bike" into "Cycling"?
Another one is with fitbits "Weights" which translates to runkeepers "Strength Training".

Is there anyway this could be implemented?  

Again thank you for your time and effort!

**Comment by fiddur** [comment]
Yeah, it's just expanding the mapping list here: https://github.com/fiddur/fitbit2runkeeper/blob/master/lib/convertFb2Hg.js#L27

Feel free to send a PR, otherwise I'll look into it in a few weeks probably...

**Comment by nemjuy** [comment]
Hi, I have the same issue: I've revoked the permission on Runkeeper and can't add the app again. Could you help me? Thanks!
